CONTRIBUTING.md

```diff
@@ -54,14 +53,30 @@ First of all, thanks for taking the time to contribute! This project can only gr
-  <li><b>plain</b>: a one-color version of the original logo.<a href="https://github.com/devicons/devicon/blob/master/icons/android/android-plain.svg"> Example </a></li>
-  <li><b>plain-wordmark</b>: a one-color version of the original logo but with wordmark.<a href="https://github.com/devicons/devicon/blob/master/icons/android/android-plain-wordmark.svg"> Example </a></li>
-  <li><b>line</b>: a one-color, line version of the original logo.<a href="https://github.com/devicons/devicon/blob/master/icons/apache/apache-line.svg"> Example </a></li>
-  <li><b>line-wordmark</b>: a one-color, line version of the original logo but with wordmark.<a href="https://github.com/devicons/devicon/blob/master/icons/apache/apache-line-wordmark.svg"> Example </a></li>
+  <li><b>original</b>: the original logo. Can contain multiple colors. Example:<br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/gimp/gimp-original.svg' height='100px' width='100px' /> <code>gimp-original.svg</code>
+
+  <li style='margin-top: 3em'><b>original-wordmark</b>: similar to the above but must contain the name of the technology. Example: <br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/gimp/gimp-original-wordmark.svg' height='100px' width='100px' /> <code>gimp-original-wordmark.svg</code>
+  </li>
+
+  <li style='margin-top: 3em'><b>plain</b>: a one-color version of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. Example: <br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/gimp/gimp-plain.svg' height='100px' width='100px' /> <code>gimp-plain-wordmark.svg</code>
+  </li>
+
+  <li style='margin-top: 3em'><b>plain-wordmark</b>: a one-color version of the original logo but with wordmark. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. Example: <br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/gimp/gimp-plain-wordmark.svg' height='100px' width='100px' /> <code>gimp-plain-wordmark.svg</code>
+  </li>
+
+  <li style='margin-top: 3em'><b>line</b>: a one-color, line version of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. Example: <br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/apache/apache-line.svg' height='100px' width='100px' /> <code>apache-line.svg</code>
+  </li>
+
+  <li style='margin-top: 3em'><b>line-wordmark</b>: a one-color, line version of the original logo but with wordmark. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. Example: <br>
+  <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/apache/apache-line-wordmark.svg' height='100px' width='100px' /> <code>apache-line-wordmark.svg</code>
+  </li>
```

Comment:
I appreciate the idea of example icons! Maybe we can use the devicon-icon as example instead of gimp and apache? We have it in all versions so I think it would be a good example. Using gimp and apache might be confusing for new contributors.

I was also wondering if a `table` instead of a unordered list (`<ul>`) would look cleaner for showing the examples?

Reply:
The table sounds like a good idea. Let me try it out.

Reply:
Hey @amacado, we unfortunately do not have `line` and `line-wordmark` versions of the `devicon` icons. What should I use for those ones instead?

Reply:
I don't know yet.. Maybe we should create a issue to come up with a `line` and `line-wordmark` version of devicon.